comp/trace/config/setup.go

```diff
@@ -134,15 +132,10 @@ func prepareConfig(c corecompcfg.Component) (*config.AgentConfig, error) {
 			cfg.RemoteConfigClient = client
 		}
 	}
-	cfg.ContainerTags = containerTagsFunc
-	cfg.ContainerProcRoot = coreConfigObject.GetString("container_proc_root")
+	containerSetup(cfg, coreConfigObject.GetString("container_proc_root"))
 	return cfg, nil
 }
 
-func containerTagsFunc(cid string) ([]string, error) {
-	return tagger.Tag("container_id://"+cid, collectors.HighCardinality)
-}
-
```

Comment:
Can we just stub the `containerTagsFunc` function? It'd be enough to isolate the tagger dependency with minimum code changes.

Reply:
✅  thanks!